frontend/src/app/modules/auth/sign-in/sign-in.component.sass

```diff
@@ -17,4 +17,4 @@
     display: grid
     grid-template-rows: repeat(3, 5vh)
     grid-template-columns: 24vw
-    gap: 2vh
\ No newline at end of file
+    gap: 3vh
```

Comment:
Can you make this gap in pixels? As on a big screen it may be too big and on phone, it may be too small